### ISSUE TITLE
sr_CS locale does not have a name and the flag is wrong

### ISSUE THREAD

**Issue by aleksmark** [Sylius/Sylius#11493]
**Sylius version affected**: 1.7.4

**Description**
I want to use Serbian Latin language in my Sylius application. In available locals, there are few Serbian locals I can use, and two in particular for Latin (sr_Latn_RS and sr_Latn).

![image](https://user-images.githubusercontent.com/14918537/82612751-89b8ca00-9bbb-11ea-8673-ead15cad3bc0.png)

However, to benefit from out of the box translation for Serbian Latin that comes with Sylius, I realized I must use sr_CS locale. That is the locale for Serbian Latin configurable through crowdin.com.  https://crowdin.com/project/sylius/sr-CS#

Notice that local does not exist in this list available locales list!

During the sylius:install, I tried to set sr_CS as my locale, which was a success, but the locale does not work correctly.

I don't see the name of the locale in list on edit channel page. 
![image](https://user-images.githubusercontent.com/14918537/82613220-ad304480-9bbc-11ea-8707-3be750c4f981.png)

On the edit product page, the flag is not correct. It is the flag od Yugoslavia, the country Serbia was part of, and that no longer exits. I believe that sr_CS was a locale used for Yugoslavia. As a name, the code is used.
![image](https://user-images.githubusercontent.com/14918537/82613568-81fa2500-9bbd-11ea-9974-3e2ea5d44b8b.png)

**More**

Prior to version 1.7.4, there was this exception on shop pages

```
An exception has been thrown during the rendering of a template ("Cannot find name for "sr_CS" locale code")
```
which was solved by this PR https://github.com/Sylius/Sylius/pull/11391

According to this comment, the issue with sr_CS exists since June 2017
https://github.com/Sylius/Sylius/issues/8086#issuecomment-306566478

**Steps to reproduce**  
do fresh sylius install with these settings
![image](https://user-images.githubusercontent.com/14918537/82613920-7bb87880-9bbe-11ea-8a76-04764c2ccb88.png)

**Possible Solution**  
I am new to Sylius and I would be happy to fix this, if I can get some guidelines on where to start.

**Comment by vvasiloi** [comment]
`sr_CS` is the code for `Serbian (Serbia and Montenegro)`.
It doesn't have a translation because it is obsolete and the flag is the right one for [Serbia and Montenegro](https://en.wikipedia.org/wiki/Serbia_and_Montenegro)
Besides that Sylius has the `sr_CS` translations in Latin script instead of Cyrillic (according to [specification](https://demo.icu-project.org/icu-bin/locexp?d_=en&_=sr_CS))
I guess Sylius should rename `sr_CS` to `sr_Latn_RS` if it's appropriate.

**Comment by aleksmark** [comment]
Yes, renaming `sr_CS` to `sr_Latn_RS` makes sense.

**Comment by vvasiloi** [comment]
@lchrusciel will renaming the translation files be enough?

**Comment by lchrusciel** [comment]
Probably a little bit more work will be needed, as flags are automatically translated from the second part of the locale. The list is generated by Symfony Bundle. However, if renaming the translation files will improve your installation, I'm open to that.

**Comment by vvasiloi** [comment]
> as flags are automatically translated from the second part of the locale

@lchrusciel that was changed more than a year ago https://github.com/Sylius/Sylius/pull/10553
I am curious how this will work with Crowdin. The target is to make `sr_CS` translations become `sr_Latn_RS`.

**Comment by lchrusciel** [comment]
Good catch Victor, I've missed this PR. Probably we should rather just create a new file, but let's summon our Crowdin Magician: @pamil

**Comment by rimas-kudelis** [comment]
Like I ranted on Slack a couple times already, Crowdin translations seem barely maintained. Just look at the screenshot of below: other then `master`, the last available version is 1.7, AND the only substantially translated version is 1.0. To an extent, same trend applies to all other locales I've checked, including `sr`, `sr_RS`, and even `pl`. I refuse to believe that 1.1 was such a big change that **none** of the translations from 1.0 could be reused. Instead. it feels as if existing translations are never really imported from Sylius to Crowdin, as if the sync is the other way only, which causes Crowdin to heavily under-report translation status and makes it difficult to contribute meaningfully. I'm pretty certain that Lithuanian translation is way more complete than the measly 85 strings reported by Crowdin.

Add to that the fact that upon contributing **each and every single string**, I'm being warned that an admin (some Polish person with zero knowledge of Lithuanian, presumably?) has to review and approve that specific submission, and contributing there becomes a very unattractive feat.

My suggestion is to quickly review all pending translations and then just reboot the whole process by first dropping all existing translations from Crowdin and then reimporting ones from the master branch ONLY. While at this, the Serbian locale discussed here could also be renamed to use the correct code.

I could probably help with that, but obviously, that would require you granting me MOAR permissions on Crowdin than I currently have as a mere translator. Plus I'd probably need a quick intro to how syncing to/from Crowdin is implemented currently, and/or at least a contact point to ping when I have questions I can't find answers to.
![paveikslas](https://user-images.githubusercontent.com/2812673/152731647-e7531502-b503-4de1-b721-51a37f70a108.png)

**Comment by vvasiloi** [comment]
I also don't like how Crowdin integration works now (it mostly doesn't work).
For example, the `sylius.ui.name` (EN: `Name`) was translated in Romanian as `Product name`, so now you see `Product name` all other the admin panel.
I took me an awful amount of time to find an change the translation in Crowdin and I don't even remember how long ago it was.
Anyway, It's still wasn't update and now I can't even find it.
